pkg/compose/build.go

```diff
@@ -370,7 +370,7 @@ func addPlatforms(project *types.Project, service types.ServiceConfig) ([]specs.
 
 	if service.Platform != "" && !utils.StringContains(service.Build.Platforms, service.Platform) {
 		if len(service.Build.Platforms) > 0 {
-			return nil, fmt.Errorf("service.platform should be part of the service.build.platforms: %q", service.Platform)
+			return nil, fmt.Errorf("service.platform %q should be part of the service.build.platforms: %q", service.Platform, service.Build.Platforms)
```

Comment:
To be coherent, we should also update the error log line 355, can you apply the same modification?

You also need to update `TestBuildPlatformsStandardErrors` end-to-end tests in the [build_test.go ](https://github.com/docker/compose/blob/v2/pkg/e2e/build_test.go) file